src/components/Dropzone/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ChangeEvent, useRef } from 'react'
```

Comment:
Let's create a `Dropzone.tsx` with all this and then inside `index.ts` export this component and all types that need to be exported (see `InputV2` as example)